add rangeInput guidence and variants (#1898)
grommet/hpe-design-system: aries-site/src/pages/components/rangeinput.mdx

import { Example } from '../../layouts';
import {
  RangeInputExample,
  RangeInputInFormExample,
  RangeInputDescriptionExample,
} from '../../examples';

<Example
  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/rangeinput/RangeInputExample.js"
  docs="https://v2.grommet.io/rangeinput?theme=hpe#props"
  figma="https://www.figma.com/file/BqCjvjc0rECQ4Ln2QhyjNi/HPE-Range-Input-Component?node-id=1%3A11"
>
  <RangeInputExample />
</Example>

## Guidance

> A RangeInput allows a user to adjust the value by either decreasing or increasing the value.

Reviewer: ```suggestion
RangeInput allows a user to easily select a value lying between a minimum and maximum range.
```

> The user can adjust the value by moving the thumb across the track.

Reviewer: ```suggestion
The value is selected by sliding a thumb control across the track.
```

- The RangeInput should be clear within the label for the user to know what value they are choosing.

> - There should be the min and the max on each side of the RangeInput so the user is aware of these values.

Reviewer: ```suggestion
- Each end of the range should be labeled by the minimum and maximum values so the user is aware of the range's possible values.
```

> - Do not use ranges that are to large i.e 1-10000 or ranges that are very small i.e 1-3

Author: I wasn't taking into account if you adjust your `step` appropriately then it shouldn't matter. I was reading a article about designing `slider` and it mentioned this but your right if the `step` is adjusted accordingly it should be fine!

> ### Usage

Reviewer: Would like to hear about more tangible use cases / applications (e.g. volume, lighting dimmer, other more enterprise focussed use cases)

Author: Ill add some more to `Usage` for this!
